autogluon/utils/tabular/ml/trainer/model_presets/presets.py

```diff
@@ -48,6 +50,7 @@
     CAT=CatboostModel,
     NN=TabularNeuralNetModel,
     LR=LinearModel,
+    Transf=TabTransformerModel,
 )
 
 DEFAULT_MODEL_NAMES = {
```

Comment:
add to DEFAULT_MODEL_NAMES:

`TabTransformerModel: 'Transformer'

Reply:
Done